units/console-getty.service.in

```diff
@@ -37,6 +36,9 @@ KillMode=process
 {% endif %}
 IgnoreSIGPIPE=no
 SendSIGHUP=yes
+# We want to do all TTY setup for container consoles ourselves, so let's prevent login and agetty from
+# touching the tty.
+CapabilityBoundingSet=~CAP_SYS_TTY_CONFIG
```

Comment:
hmm, this will of course trickle down into user the session and from there into `screen` stuff too, or into `minicom`. I wonder if people might be upset if we take the ability to reconfigure *any* tty from them.

what kind of set up does agetty still do once we use --keep-baud? Wouldn't it be a better idea to maybe ask @karelzak to add more switches like that to agetty?

Reply:
It's not just about the tty settings though, it's also about the unconditional vhangup() done by login that we can't control even if TTYVHanup is disabled on systemd's side. But this is probably not the right way to fix that anyway.

Reply:
i am surprised it does that, given there's this stuff in the agetty man page:

```
       -R, --hangup
           Call vhangup(2) to do a virtual hangup of the specified terminal.

```

Reply:
Or isn't this a bug in agetty, if it does a vhangup, even if --hangup is not specified?

Reply:
Agetty doesn't do a vhangup unconditionally, but it execs login, and that does an unconditional vhangup

Reply:
hmm, but that sounds like something to fix in login? 

any idea why both login and getty feel the need to issue vhangup()?

@karelzak any idea about all this? can we get rid of the extra vhangup in login?